doc/get-set-friendlyname-mac.md

```diff
@@ -58,45 +58,52 @@ Use the FriendlyName device resource to retrieve `FriendlyName` of a camera.
 ```shell
 curl http://localhost:59882/api/v2/device/name/<device name>/FriendlyName | jq .
 ```
-2. Response from the curl command. FriendlyName value can be found under `objectValue` in the response.
+2. Response from the curl command. FriendlyName value can be found under `value` field in the json response.
 ```shell
 {
-   "apiVersion" : "v2",
-   "event" : {
-      "apiVersion" : "v2",
-      "deviceName" : "TP-Link-C200-3fa1fe68-b915-4053-a3e1-cc32e5000688",
-      "id" : "a96e6fb1-d61c-49bf-b243-6e7b7db579d0",
-      "origin" : 1657921895566531752,
-      "profileName" : "onvif-camera",
-      "readings" : [
-         {
-            "deviceName" : "TP-Link-C200-3fa1fe68-b915-4053-a3e1-cc32e5000688",
-            "id" : "0cc6ae1f-1755-45ef-8a62-29bb69b3b821",
-            "objectValue" : "Home camera",
-            "origin" : 1657921895566531752,
-            "profileName" : "onvif-camera",
-            "resourceName" : "FriendlyName",
-            "value" : "",
-            "valueType" : "Object"
-         }
-      ],
-      "sourceName" : "FriendlyName"
-   },
-   "statusCode" : 200
+  "apiVersion": "v2",
+  "statusCode": 200,
+  "event": {
+    "apiVersion": "v2",
+    "id": "5b924351-31c7-469e-a9ba-dea063fdbf3a",
+    "deviceName": "TP-Link-C200-3fa1fe68-b915-4053-a3e1-cc32e5000688",
+    "profileName": "onvif-camera",
+    "sourceName": "FriendlyName",
+    "origin": 1658441317910501400,
+    "readings": [
+      {
+        "id": "62a0424b-a3c1-45ea-b640-58c7aa3ea476",
+        "origin": 1658441317910501400,
+        "deviceName": "TP-Link-C200-3fa1fe68-b915-4053-a3e1-cc32e5000688",
+        "resourceName": "FriendlyName",
+        "profileName": "onvif-camera",
+        "valueType": "String",
+        "value": "Home camera"
+      }
+    ]
+  }
 }
 ```
+
+## Preset MACAddress
+`MACAddress` is an element in the `Onvif ProtocolProperties` device field. It will be set to empty string if no value is provided, or
+it will be set with the MAC address value of the camera if valid credentials are provided.
+The user can pre-define this field in a camera.toml file.
+
+If you add pre-defined devices, set up the `MACAddress` field as shown in the
+[camera.toml.example file](cmd/res/devices/camera.toml.example).
+
 ## Set MAC Address
 
-Use the device resource MACAddress to set `MACAddress` of a camera.
+MACAddress can also be set via Edgex device command.
+Device resource MACAddress is used to set `MACAddress` of a camera.
```

Comment:
I would add the following info after yours.
```markdown
This is useful for setting the MAC Address for devices which do not contain the MAC Address in the Endpoint Reference Address, or have been added manually without a MAC Address. Since the MAC is used to map credentials for cameras, it is important to have this field filled out.

> Note: When a camera successfully becomes `UpWithAuth`, the MAC Address is automatically queried and overriden by the system if available.
```